dev-docker-compose.yml

```diff
@@ -7,3 +7,7 @@ services:
     environment:
       - DEBUG=*
       - arangodb__ensureDefaults="true"
+
+  arangodb:
+    ports:
+      - 8529:8529
```

Comment:
@awlayton This seems safe I think? If we keep the Azure firewall for 8529 closed, then we can simply access over ssh port forward.